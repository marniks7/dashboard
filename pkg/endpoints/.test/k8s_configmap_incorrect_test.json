{
    "apiVersion": "v1",
    "data": {
        "version": "v0.39.0"
    },
    "kind": "ConfigMap",
    "metadata": []
}
