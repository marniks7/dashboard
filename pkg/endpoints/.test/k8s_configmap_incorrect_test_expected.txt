json: cannot unmarshal array into Go struct field K8sItem.metadata of type endpoints.K8sObjectMeta
